py_gtfs_rt_ingestion/ingest.py

```diff
@@ -1,78 +1,86 @@
+#!/usr/bin/env python
+
 import argparse
 import gzip
 import json
 import sys
 
-from pathlib import Path
 import pyarrow as pa
 import pyarrow.parquet as pq
 
-from py_gtfs_rt_ingestion.helpers import get_datetime_from_header
-from py_gtfs_rt_ingestion.helpers import get_vehicle_schema
+from datetime import datetime
+from pathlib import Path
 
-DESCRIPTION = ""
+from py_gtfs_rt_ingestion import Configuration
 
+DESCRIPTION = "Convert a json file into a parquet file. Used for testing."
 
 def parseArgs(args) -> dict:
     parser = argparse.ArgumentParser(description=DESCRIPTION)
     parser.add_argument(
-        '--fname',
-        dest='fname',
+        '--input',
+        dest='input_file',
         type=str,
         required=True,
         help='provide filename to ingest')
 
+    parser.add_argument(
+        '--output',
+        dest='output_dir',
+        type=str,
+        required=True,
+        help='provide a directory to output')
+
     parsed_args = parser.parse_args(args)
 
     return parsed_args
 
 
-def run(filename) -> None:
-    if 'https_cdn.mbta.com_realtime_VehiclePositions_enhanced' not in filename:
-        raise Exception('Requires VehiclePostitions file.')
+def convert_json_to_parquet(input_filename: str, output_dir: str) -> None:
+    """
+    convert in input *.json.gz file into a parquet file
 
-    fname = Path(filename)
-    schema = get_vehicle_schema()
+    * get the configuration information to transform
+    * create a table object
+    * parse datetime from the header of the json file
+    * append to the table for each element the entities list in the json file
+    * write the table
+    """
+    config = Configuration.from_filename(input_filename)
 
-    pa_table_dict = {k.name:[] for k in schema}
+    schema = config.get_schema()
+    table = {key.name:[] for key in schema}
```

Comment:
get the config from filename (which raises an exception on bad filenames) and get the schema from the configuration.